project/src/main/webapp/script.js

```diff
@@ -62,11 +121,131 @@ function createPlaceElement(place) {
 }
 
 /**
- * Displays the query form to the user and hides the results, so that the user can try again with a
- * different query.
+ * Displays the query form to the user and hides the results, so that the user can try to get new
+ * results.
  */
 function tryAgain() {
   document.getElementById('query-form').style.display = 'block';
   document.getElementById('results').style.display = 'none';
   document.getElementById('place').innerHTML = '';
 }
+
+function addSearchBoxToMap(map, input) {
+  // Create the search box and link it to the UI element.
+  const searchBox = new window.google.maps.places.SearchBox(input);
+  map.controls[window.google.maps.ControlPosition.TOP_LEFT].push(input);
+  // Bias the SearchBox results towards current map's viewport.
+  map.addListener("bounds_changed", () => {
+    searchBox.setBounds(map.getBounds());
+  });
+  let markers = [];
+  // Listen for the event fired when the user selects a prediction and retrieve
```

Comment:
What does it mean "the user enters prediction" ?

Reply:
The query he enters to the search box. I changed the documentation